pyang/grammar.py

```diff
@@ -546,10 +546,10 @@ def is_rule_less_than(ra, rb):
 Maps a statement name to a 2-tuple:
     (<argument type name> | None, <list of substatements> )
 Each substatement is a 2-tuple:
-    (<statement name>, <occurance>) |
+    (<statement name>, <occurence>) |
```

Comment:
The correct spelling should be `occurrence`, refer to https://www.gingersoftware.com/english-online/spelling-book/misspelling/occurrence-occurance-occurence